overlay.d/07c9s/bootstrap/pre-pivot.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+set -euxo pipefail
+
+# This script is called by the OKD installer and runs in the
+# local context of the bootstrap node.
+# It rebases the booted operating system (e.g. Fedora CoreOS)
+# to CentOS Stream CoreOS 9
+
+# Load common functions
+. /usr/local/bin/bootstrap-service-record.sh
+. /usr/local/bin/release-image.sh
+
+record_service_stage_start "rebase-to-scos"
+
+# Copy manifests
+# Before rebasing an FCOS node to SCOS, ensure the OKD manifests are in place
+if [ ! -d /opt/openshift/openshift/ ]; then
+    mkdir -p /opt/openshift/openshift/
+fi
+cp -irvf manifests/* /opt/openshift/openshift/
+
+# Disable all yum repos
+sed -i 's/^enabled=1/enabled=0/g' /etc/yum.repos.d/*.repo
+
+# Pivot to new ostree container image
+OS_IMAGE=$(image_for centos-stream-coreos-9)
+rpm-ostree rebase --experimental "ostree-unverified-registry:${OS_IMAGE}"
+
+# Delete cached rpm repo metadata
+rpm-ostree cleanup -m
+
+touch /opt/openshift/.pivot-done
```

Comment:
@travier this stamp file is an (undocumented) OKD installer API: https://github.com/openshift/installer/blob/d10d64c9843e37f500bcb8d66caf339e940b918f/data/data/bootstrap/files/usr/local/bin/bootstrap-pivot.sh#L10

This script will only be run if the file doesn't already exist.